bot/cogs/last_fm.py

```diff
@@ -1,14 +1,14 @@
-import discord
+import disnake
 import urllib
 import aiohttp
-from discord.ext import commands
+from disnake.ext import commands
 from bot.settings import MUSIC_TOKEN
 
 
 class Music(commands.Cog):
     """Commands for song searching."""
 
-    def __init__(self, bot: discord.ext.commands.bot.Bot):
+    def __init__(self, bot: disnake.ext.commands.bot.Bot):
```

Comment:
Not an issue that was made by this PR, but we can deal with this now, since we're here

```suggestion
    def __init__(self, bot: commands.Bot):
```